{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -13,7 +13,6 @@ classifiers = [
   "Operating System :: POSIX :: Linux",
   "Operating System :: MacOS :: MacOS X",
   "Operating System :: Microsoft :: Windows",
-  "Programming Language :: Python :: 3",
```

Comment:
Not sure what this was for. Have removed because it cannot be parsed like the others :-)